src/components/PriceChange.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React, { Component } from "react";
+import LText from "./LText";
+import colors from "../colors";
+
+export default class PriceChange extends Component<{
+  before: number,
+  after: number,
+  color?: "western" | "eastern",
+  style?: {}
+}> {
+  render() {
+    const { before, after, color } = this.props;
+    const style: { color?: * } = {};
+
+    if (!before) {
+      return <LText />;
+    }
+
+    const percent: number = after / before * 100 - 100;
+
+    if (color === "western") {
+      style.color = percent >= 0 ? colors.green : colors.red;
+    } else if (color === "eastern") {
+      style.color = percent >= 0 ? colors.red : colors.blue;
+    }
+
+    return (
+      <LText style={[this.props.style, style]}>
```

Comment:
lil' detail here, but reverse style & this.props.style IMO. (basically we should allow style to override color if needed)